valhalla/baldr/openlr.h

```diff
@@ -161,25 +161,117 @@ struct LocationReferencePoint {
   FormOfWay fow;        // 5.2.3. Form of way
 };
 
+// 5.3.1
+// The side of road information (SOR) describes the relationship between the
+// point of interest and a referenced line. The point can be on the right
+// side of the line, on the left side of the line, on both sides of
+// the line or directly on the line.
+enum SideOfTheRoad {
+  DirectlyOnRoadOrNotApplicable = 0,
+  RightSideOfRoad = 1,
+  LeftSideOfRoad = 2,
+  BothSidesOfRoad = 3,
+};
+
+// 5.3.2
+// The orientation information (ORI) describes the relationship between the point
+// of interest and the direction of a referenced line. The point may be directed
+// in the same direction as the line, against that direction, both directions,
+// or the direction of the point might be unknown.
+enum Orientation {
+  NoOrientation = 0,
+  FirstLrpTowardsSecond = 1,
+  SecondLrpTowardsFirst = 2,
+  BothDirections = 3,
+};
+
+const uint8_t OPENLR_VERSION = 3;
+
+// The first byte of the OpenLr identifier, page 55
+struct OpenLrStatus {
```

Comment:
thanks for structifying this! we should do the other attributes at some point but this is a great start!